src/Sulu/Bundle/MediaBundle/Controller/MediaPreviewController.php

```diff
@@ -94,10 +94,11 @@ public function postAction($id, Request $request)
             $previewImage = $this->mediaManager->save($uploadedFile, $data, $this->getUser()->getId());
 
             $mediaEntity->setPreviewImage($previewImage->getEntity());
+            $this->mediaManager->addFormatsAndUrl($media);
 
             $this->entityManager->flush();
 
-            $view = $this->view($previewImage, 200);
+            $view = $this->view($media, 200);
```

Comment:
i guess this controller is not tested anywhere as you did not adjust any test-case related to this?

Reply:
Yeah, you are right... I think I am going to add a test case.

Reply:
And btw, this is the reason https://github.com/sulu/sulu/pull/4918/files#r353064765

Reply:
perfect!